conf/messages.en

```diff
@@ -57,6 +58,20 @@ date_of_death.error = Give a correct date of death
 date_of_death.error.day_invalid = Give a correct day of death using numbers 1 to 31
 date_of_death.error.month_invalid = Give a correct month of death using numbers 1 to 12
 date_of_death.error.year_invalid = Give a correct year of death
+date_Of_death.error.date_not_complete = Give a full date of death
```

Comment:
Need to get the Welsh translations for new content - email gareth.jones4@hmrc.gsi.gov.uk